.ruby-version

```diff
@@ -1 +1 @@
-2.6.6
+2.7.0
```

Comment:
I don't think we want to have these changes in the project, this is going to cause issues with the deployment. We need to stay on the same version as before.